rclc_examples/src/example_executor_only_rcl.c

```diff
@@ -83,6 +85,13 @@ int main(int argc, const char * argv[])
     printf("Error in rcl_node_init\n");
     return -1;
   }
+  if (rcl_logging_rosout_enabled() && node_ops.enable_rosout) {
+    rc = rcl_logging_rosout_init_publisher_for_node(&my_node);
+    if (rc != RCL_RET_OK) {
+      printf("Error in rcl_logging_rosout_init_publisher_for_node\n");
```

Comment:
since this is an example, i think it would be okay not to call `rcl_logging_rosout_init_publisher_for_node`.